spec/models/subheading_spec.rb

```diff
@@ -86,9 +86,26 @@
     end
   end
 
-  describe 'declarable?' do
+  describe '#declarable?' do
     subject { heading.declarable? }
 
     it { is_expected.to be false }
   end
+
+  describe '#ancestors' do
+    subject { subheading.ancestors.map(&:goods_nomenclature_sid) }
+
+    let :subheading do
+      build :subheading, goods_nomenclature_sid: commodities[3][:goods_nomenclature_sid],
+                         commodities:
+    end
+
+    let :commodities do
+      attributes_for_list :commodity, 5 do |c, i|
+        c[:parent_sid] = (i > 0 ? c[:goods_nomenclature_sid] - 1 : nil)
```

Comment:
There's a lot of baked in programmatic understanding in the fixtures, here. Is there any chance we could just be explicit with some sids to simulate real data/also make the fixtures a bit more dumb?

Reply:
Problem with hard coding sid's (or any ids) is that you can only use the factory once per spec (you couldn't create 2 subheadings with tree's of commodities under them for example).

Reply:
I'm updating this to use manually specified ids but since I'm having to specify more ids I'm pretty sceptical that it _reduces_ the knowledge subheading has of commodity tree's - it seems to result in an increase